lib/ntc_templates/parse.py

```diff
@@ -12,11 +12,13 @@
 
 
 def _get_template_dir():
-    package_dir = os.path.dirname(__file__)
-    template_dir = os.path.join(package_dir, "templates")
-    if not os.path.isdir(template_dir):
-        project_dir = os.path.dirname(os.path.dirname(os.path.dirname(template_dir)))
-        template_dir = os.path.join(project_dir, "templates")
+    template_dir = os.environ.get("NTC_TEMPLATES_DIR")
```

Comment:
I'm wondering if we should offer both an OS environment or pass in as an arg to `parse_output`.

Something like.

```
def parse_output(platform=None, command=None, data=None, template_dir=None):
    template_dir = os.environ.get("NTC_TEMPLATES_DIR") or template_dir
    if template_dir is None:
        template_dir = _get_template_dir()
```

And then keep `_get_template_dir()` the same.

Reply:
I would say it is good to have `_get_template_dir` do the environment lookup, but allow this function to override the environment by passing in the directory.

```
def parse_output(platform=None, command=None, data=None, template_dir=None):
    template_dir = template_dir or _get_template_dir()
```

I'm also okay with the PR like it is since that is more than is available today and a reasonable solution

Reply:
@jmcgill298 @FragmentedPacket , just let me know either way.  I don't mind taking some extra time to implement the ability to set an arg.

Reply:
I'm with @jmcgill298 and his way is better than mine.

I'll approve it and let you decide :)